presets/index.js

```diff
@@ -15,9 +15,9 @@ module.exports = (options = {}) => {
     alchemyGoerli: [`wss://eth-goerli.ws.alchemyapi.io/v2/${options.alchemyId}`, `https://eth-goerli.alchemyapi.io/v2/${options.alchemyId}`],
     infuraPolygon: [`wss://polygon-mainnet.infura.io/ws/v3/${options.infuraId}`, `https://polygon-mainnet.infura.io/v3/${options.infuraId}`],
     infuraArbitrum: [`wss://arbitrum-mainnet.infura.io/ws/v3/${options.infuraId}`, `https://arbitrum-mainnet.infura.io/v3/${options.infuraId}`],
-    infuraOptimism: [`wss://optimism-mainnet.infura.io/ws/v3/${options.infuraId}`, `https://optimism-mainnet.infura.io/v3/${options.infuraId}`],
-    idChain: ['wss://idchain.one/ws/'],
-    gnosis: ['https://rpc.gnosischain.com '],
+    infuraOptimism: [`https://optimism-mainnet.infura.io/v3/${options.infuraId}`],
```

Comment:
does Infura no longer have have a ws endpoint for Optimism?

Reply:
No it doesn't work - this has been https only in Frame for a while:

https://github.com/floating/frame/pull/453